core/interpreter/src/main/java/org/overture/interpreter/runtime/ModuleInterpreter.java

```diff
@@ -278,8 +283,12 @@ public Value execute(String line, DBGPReader dbgp) throws Exception
 		MainThread main = new MainThread(expr, mainContext);
 		main.start();
 		scheduler.start(main);
+		Value result = main.getResult();
+
+		EvaluationCache.getInstance().logCacheStats();
+		EvaluationProfiler.getInstance().logResults();
 
-		return main.getResult(); // Can throw ContextException
+		return result; // Can throw ContextException
```

Comment:
⛏️ The comment should have been on line 286, since it's the function call that can throw an exception, but it doesn't really matter if we're not upstreaming this.